pkgs/development/python-modules/withings-api/default.nix

```
@@ -0,0 +1,54 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, poetry-core
+, arrow
+, requests-oauthlib
+, typing-extensions
+, pydantic
+, unittestCheckHook
+, pytest
+, responses
+}:
+
+buildPythonPackage rec {
+  pname = "withings-api";
+  version = "2.4.0";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "vangorra";
+    repo = "python_withings_api";
+    rev = "refs/tags/${version}";
+    sha256 = "sha256-8cOLHYnodPGk1b1n6xbVyW2iju3cG6MgnzYTKDsP/nw=";
+  };
+
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace 'requests-oauth = ">=0.4.1"' '''
+  '';
+
+  propagatedBuildInputs = [
+    poetry-core
+    arrow
+    requests-oauthlib
+    typing-extensions
+    pydantic
+  ];
+
+  checkInputs = [
+    unittestCheckHook
+    pytest
```

```suggestion
    pytestCheckHook
```

If you want to use pytest, use the appropriate hook. You'll also notice we need to drop the coverage arguments since we don't want to run coverage, which is meaningless to us.

---

Also unittestCheckHook did not find the tests, so none were run.

---

Can you either do this as your own commit or point me towards where to find how to drop coverage arguments?

---

https://github.com/vangorra/python_withings_api/blob/master/pyproject.toml#L99

---

Solved! Sorry. Didn't notice that.